src/System.Net.WebSockets.Client/tests/ClientWebSocketOptionsTests.cs

```diff
@@ -18,6 +18,12 @@ namespace System.Net.WebSockets.Client.Tests
 {
     public partial class ClientWebSocketOptionsTests : ClientWebSocketTestBase
     {
+        // Recreate the minimum WebSocket buffer size values from the Framework version of WebSocket.
+        // We need these values to test when the minimums are violated, but the values set in Framework
+        // are internal to the WebSocketBuffer class.
+        private const int MinSendBufferSize = 16;
+        private const int MinReceiveBufferSize = 256;
+
```

Comment:
You're right, I think that makes the code a lot cleaner. I'll go ahead and make those changes.